### ISSUE TITLE
about loss

### ISSUE THREAD

**Issue by qianyewu** [ANLGBOY/SoftFlow#1]
Hello, I would like to ask about the loss. I have been training for a long time, but the loss has always been large, which is maintained at about 30000.
Have I not been training long enough?Finally, after you've trained 15,000 EPOCHs, what is loss going to be？THANKS

**Comment by ANLGBOY** [comment]
![validation_loss_curve](https://user-images.githubusercontent.com/40254383/112775855-0ab4dd80-9079-11eb-9aa3-6528f71f89c8.png)

**Comment by ANLGBOY** [comment]
The above picture shows the validation loss curves when we trained the models.

**Comment by lx709** [comment]
Same problem. I guess maybe we can remove the constant value in "standard_normal_logprob" function as PointFlow's code. This will give smaller prior prior_nats.

--------------------
def standard_normal_logprob(z):
    dim = z.size(-1)
    log_z = -0.5 * dim * log(2 * pi)
#     return log_z - z.pow(2) / 2
    return - z.pow(2) / 2

**Comment by lx709** [comment]
@ANLGBOY @qianyewu How long does it take to train the model?

**Comment by ANLGBOY** [comment]
@lx709 It took almost 5~6 days with four 2080Ti GPUs for modeling each shape!
